tasks/extract_filepaths_for_granule/schemas/config.json

```diff
@@ -5,110 +5,24 @@
     "type": "object",
     "properties": {
       "buckets": {
-        "description": "The list of dictionary having bucket name and type",
-        "type": "array",
-        "items": [
-          {
+        "type": "object",
+        "description": "AWS S3 buckets used by this task",
```

Comment:
Possibly update this description to be more helpful instead of the cumulus version. Maybe something like.
`AWS S3 bucket mapping used for Cumulus and ORCA configuration.`

Reply:
updated